go/ephemeral/user_ek_box_storage.go

```diff
@@ -109,7 +109,8 @@ func (s *UserEKBoxStorage) Get(ctx context.Context, generation keybase1.EkGenera
 
 	defer s.Unlock() // release the lock after we unbox
 	if cacheItem.HasError() {
-		return userEK, cacheItem.Error()
+		err = cacheItem.Error()
```

Comment:
What is going on here?

Reply:
this will allow `err` to hold the correct value for logs in the `Trace`

Reply:
If you return it, `err` gets set, so I don't think you need to do this.